contrib/docs/tutorials/authenticate-api-requests.md

```diff
@@ -73,10 +80,10 @@ export const createAuthMiddleware = async (
           cookieDomain,
         });
       }
-      next();
     } catch (error) {
       res.status(401).send('Unauthorized');
     }
+    next();
```

Comment:
Maybe add a return statement after the line `84` instead?

Reply:
I'm not sure we want to move the `next()` outside the catch block. With this change the you would send Unauthorised and then call the next middleware which might lead to problems if that one also tries calling `res.send` for example. Or have I misunderstood something here @matchaxnb?